modules/social_features/social_follow_user/social_follow_user.links.menu.yml

```
@@ -3,3 +3,9 @@ social_follow_user.settings:
   description: 'Enable module functionality.'
   route_name: social_follow_user.settings
   parent: social_core.admin.config.social
+
+social_follow_user.social_core.dashboard.configuration.settings:
+  title: 'Follow user settings'
+  description: 'Enable module functionality.'
```

```suggestion
  description: 'Enable functionality.'
```

Sounds more user friendly, module doesn't really say much to our SM 😊